app/UI/app.tsx

```diff
@@ -709,17 +709,22 @@ const sync_space_members = async (
     database: Database_View,
 ) => {
     for (const relay of pool.getRelays()) {
-        forever((async () => {
+        (async () => {
             const chan = relay.getSpaceMembersStream();
             for await (const spaceMembers of chan) {
                 if (spaceMembers instanceof Error) {
+                    if (spaceMembers instanceof TypeError) {
```

Comment:
When the fetch result is not json, a `SyntaxError` is reported. Chan may need to be turned off as well.
Example: https://nos.lol/api/members